### PR TITLE
chore: minor improvement to the kibana_alerting_rule docs

### PR DESCRIPTION
**Pull request by jsangilve**
Minor improvement to the new `kibana_alerting_rule` documentation specifying how to get the valid `rule_types`.

### CODE REVIEW

File: docs/resources/kibana_alerting_rule.md [hunk 1]
```diff
@@ -50,8 +51,8 @@ resource "elasticstack_kibana_alerting_rule" "example" {
 - `name` (String) The name of the rule. While this name does not have to be unique, a distinctive name can help you identify a rule.
 - `notify_when` (String) Defines how often alerts generate actions. One of `onActionGroupChange`, `onActiveAlert`, or `onThrottleInterval`
 - `params` (String) The parameters to pass to the rule type executor params value. This will also validate against the rule type params validator, if defined.
-- `rule_type_id` (String) The ID of the rule type that you want to call when the rule is scheduled to run. For more information, refer to Rule types documentation (https://www.elastic.co/guide/en/kibana/master/rule-types.html).
-
+- `rule_type_id` (String) The ID of the rule type that you want to call when the rule is scheduled to run. For more information about the valid values, list the rule types using [Get rule types API](https://www.elastic.co/guide/en/kibana/master/list-rule-types-api.html) or refer to the [Rule types documentation](https://www.elastic.co/guide/en/kibana/master/rule-types.html).
```
Comment (tobio) [inline docs/resources/kibana_alerting_rule.md:53]: These docs are generated, you'll have to make this change to the [source schema](https://github.com/elastic/terraform-provider-elasticstack/blob/main/internal/kibana/alerting.go#L57)

**Review by tobio — COMMENTED** [review]
Thanks for taking the time to improve the docs. Small change required here.

File: docs/resources/kibana_alerting_rule.md (hunk 1 above)
Comment (jsangilve) [reply docs/resources/kibana_alerting_rule.md:53]: oh my bad, I just updated this 💪

**Review by tobio — APPROVED** [review]

**Comment by tobio** [comment]
❤️ seeing contributions like this 🎉

**Pull request merged**